pkg/volume/rbd/rbd.go

```diff
@@ -650,6 +791,147 @@ func (c *rbdUnmounter) TearDownAt(dir string) error {
 	return nil
 }
 
+// <><><>
+var _ volume.BlockVolumeMapper = &rbdDiskMapper{}
+
+type rbdDiskMapper struct {
+	*rbd
+	// capitalized so they can be exported in persistRBD()
```

Comment:
this not needed any more

Reply:
ping @sbezverk

Reply:
@sbezverk the following members (mon, id, mounter, etc) are already in `rbd struct`

Reply:
done, it seemed only mounter was a redundant element existing in rbd and mapper structs. Removed from mapper struct.